pyston/data_processor.py

```diff
@@ -1,37 +1,37 @@
 from __future__ import unicode_literals
 
-import sys
 import base64
+import binascii
 import inspect
 import mimetypes
+import sys
 
+from requests.exceptions import RequestException
 from six import BytesIO
 
-from django.forms.fields import FileField
```

Comment:
opět se ti nějak rozházeli importy, použil jsi isort že? Zde není nastaven bohužel

Reply:
divne ze se to rozhazelo, pouzivam vyhradne isort s parametry -e -m 4 -w 120 $FilePath$ , pouzivam to blbe?

Reply:
většinou se pořadí knihoven které to nezná musí definovat v konfiguračním souboru, což bude ten problém